pandas/_libs/internals.pyi

```diff
@@ -50,10 +50,12 @@ class SharedBlock:
 
 class NumpyBlock(SharedBlock):
     values: np.ndarray
+    # @final  # not useful in cython, but we _would_ annotate with @final
```

Comment:
@jbrockmendel i've moved the comment from the cython since the final decorator could be added to the stub so the comment is not relevant.

However, we can't add to the stub as it is overridden in subclass in pandas/core/internals/blocks.py

Reply:
sounds like it is inaccurate so should be removed anyway